src/util/languageUtils.js

```diff
@@ -40,7 +40,8 @@ export const getLanguage = (
 	defaultLang = LANG_DEFAULT
 ) => {
 	// return the first language hit in the order of preference
-	supportedLangs.sort(l => l !== defaultLang);
+	// TODO - better comment
```

Comment:
yeah, for our apps `defaultLang` is always `en-US`, but the option of passing in a different default language makes this function more generic, and if I remember right the unit tests supply different default languages to test the behavior.

Eventually, when Meetup moves HQ to New Zealand, we'll want to update this to `en-NZ`, but it's not critical.